src/components/Slider/Slider.js

```diff
@@ -13,7 +12,7 @@ import Pit from './Pit.js';
 import autoHideContainerHOC from '../../decorators/autoHideContainer.js';
 import headerFooterHOC from '../../decorators/headerFooter.js';
 
-class Slider extends Component {
+export class Slider extends Component {
```

Comment:
Did you need this in order to query the Rheostat component itself in the tests?

Reply:
Not really but it's more easy to do with the export. It's avoid something like:

```js
shallow(<Enhance {...props} />)
    .find('Slider-HeaderFooter')
    .shallow()
    .find('Slider')
    .shallow()
    .find('Rheostat')
    .simulate('change', {
      values: [0, 100],
    });
```

Reply:
That's something we do for other components. We've prefixed the component without the wrappers with `Raw` like ClearAll (the one with the headers and autohidecontainer) and RawClearAll (the one without).